MYshort.bib

```diff
@@ -45,43 +51,54 @@ @String{ICST
 @String{ICTCS	= "ICTCS"},
 @String{ICTSS = "ICTSS"}
 @String{IDPT = "IDPT"}
+@String{IFAC = "IFAC"}
 @String{iFM = "iFM"}
 @String{IPISE = "IPISE"}
+@String{Informatik = "Jahrestagung der Gesellschaft f{\"{u}}r Informatik"}
 @String{ITNG = "ITNG"}
+@String{ITP = "ITP"}
 @String{JMLC = "JMLC"}
-@String{MSR="MSR"}
-@String{NaBIC="NaBIC"}
+@String{MSR = "MSR"}
+@String{NaBIC = "NaBIC"}
 @String{NATOSE = "NATO SE"}
 @String{NFM = "NFM"}
 @String{MODELS = "MODELS"}
+@String{Modellierung = "Proc.\ Modellierung"}
 @String{MODELSWARD = "MODELSWARD"}
 @String{MODULARITY = "Modularity"}
 @String{NODe = "NODe"},
 @String{OOPSLA = "OOPSLA"}
+@String{PCS = "PCS"}
 @String{PLDI = "PLDI"}
 @String{POPL = "POPL"}
-@String{Promise="Promise"}
-@String{QSIC="QSIC"}
+@String{PPPJ = "PPPJ"}
+@String{Promise = "Promise"}
+@String{QSIC = "QSIC"}
 @String{RAMiCS = "RAMiCS"}
+@String{RAMICS = "RAMiCS"}
 @String{RE = "RE"}
 @String{REFLECTION = "REFLECTION"}
 @String{REFSQ = "REFSQ"}
 @String{RISE = "RISE"}
 @String{RV = "RV"}
-@String{SANER = "SANER"}
 @String{SE = "SE"}
+@String{SANER = "SANER"}
+@String{SCAM = "SCAM"}
 @String{SEKE = "SEKE"}
 @String{SERENE = "SERENE"}
 @String{SERP = "SERP"}
 @String{SETSS = "SETSS"}
 @String{SIGMOD = "SIGMOD"}
 @String{SLE = "SLE"}
+@String{SOFTENG = "SOFTENG"}
 @String{SOFSEM = "SOFSEM"}
 @String{SPLC = "SPLC"}
 @String{SPLASH = "SPLASH"}
 @String{TAP = "TAP"}
 @String{TOOLSeurope = "TOOLS Europe"}
 @String{TOOLSpacific = "TOOLS Pacific"}
+@String{XPS = "XPS"}
+@String{VaMoS20 = "VaMoS"}
```

Comment:
Why is there not entry vor VaMoS then?

Reply:
Vamos20 is a conference, but Vamos is a workshop. So Vamos is already in this file but under "workshops" instead of "conferences".